syntax = "proto3";

message NameFile {
     string name_file = 1;
}

message Field {
     string field = 1;
}

message FileReturn {
     string value = 1;
}

message ValueReturn {
     float value = 1;
}

service DaskFunctions {
    rpc ReadCSV(NameFile) returns (FileReturn) {}
    rpc Max(Field) returns (ValueReturn) {}
    rpc Min(Field) returns (ValueReturn) {}
}